frameworks/kafka/docs/api-reference.md

```diff
@@ -400,7 +400,16 @@ Note the warning in the output from the commands above. You can change the indic
     {
         "message": ""
     }
-    
+
+# Service Status
```

Comment:
I'm adding this to all infinity services with a suppressed endpoint -- it doesn't look at the moment like the APIs for the various DC/OS commons services are identical. If that's the plan, let's talk about how and where to put central API docs.